### ISSUE TITLE
Do you use ReLu instead of Softmax?

### ISSUE THREAD

**Issue by headscott** [FPNAS/LBGAT#12]
In your paper I saw, that you used Softmax, but I can't find it in your code. I only see ReLu. Did I unsterstand that wrong, or don't you even use Softmax?

**Comment by headscott** [comment]
Oh wait, an Update: I found Softmax. But you only use it for loss functions, right? Not as activation function?
